lgc/state/PipelineState.cpp

```diff
@@ -1323,6 +1323,23 @@ StringRef PipelineState::getBuiltInName(BuiltInKind builtIn) {
   }
 }
 
+// =====================================================================================================================
+// Set the packable state of generic input/output
+//
+void PipelineState::setInOutPackState() {
```

Comment:
`setInOutPackState -> initializeInOutPackState`
In the future, if we could dynamically adjust the settings, we need `setInOutPackState `